packages/web-components/src/components/cbp-tooltip/cbp-tooltip.stories.tsx

```diff
@@ -56,9 +54,45 @@ Tooltip.args = {
     alignment: 'top-left',
     title: 'Test Tooltip Title',
     content: 'Stub text for tooltip.',
-    buttonContent: '<cbp-icon name="user"></cbp-icon>',
-    // sx:{
-    //     "margin-left":"40%",
-    //     "margin-top":"20%"
-    // }   
+    tooltipControl: '<cbp-icon name="user"></cbp-icon>',
+    sx:{
```

Comment:
you don't want this in your story (I was wondering why the `sx` controls was reordered higher in the list). You can center your story with: 

```
 parameters: {
    layout: 'centered',
  },
```

(will apply to both stories)